subsys/jaxrs/src/main/java/org/commonjava/indy/bind/jaxrs/util/ResponseUtils.java

```diff
@@ -465,7 +465,14 @@ else if ( ( error instanceof IndyWorkflowException ) && ( (IndyWorkflowException
             code = Status.fromStatusCode( sc );
         }
 
-        LOGGER.error( "Sending error response: {} {}\n{}", code.getStatusCode(), code.getReasonPhrase(), msg );
+        if ( code.getStatusCode() != ApplicationStatus.NOT_FOUND.code() )
```

Comment:
When we looked in our log aggregator at level == ERROR, most of what we saw was notices that we were responding with 404 to some request. This is not really an error for Indy (it's a normal use case), so I'm switching that over to debug log level. I'll probably file a follow-up commit that moves non-404 errors to use the warn log level, just to further reduce pollution of the logs for normal use cases...or maybe only if the status code is < 500.